Description: dlopen the sonamed libs directly, like it has already been done
  for libGL.so.1, as we don't want to Depend on the -dev packages for the .so files:
  - libosmesa.so.6: was already available in Hardy
  - libGLESv2.so.1 and libEGL.so.1: available only since Maverick

Index: dev.saucy/src/ui/gl/gl_implementation_x11.cc
===================================================================
--- dev.saucy.orig/src/ui/gl/gl_implementation_x11.cc	2013-05-22 16:33:48.115517978 -0500
+++ dev.saucy/src/ui/gl/gl_implementation_x11.cc	2013-05-22 16:33:48.103517562 -0500
@@ -79,7 +79,7 @@
       }
 
       base::NativeLibrary library = LoadLibrary(
-          module_path.Append("libosmesa.so"));
+          module_path.Append("libosmesa.so.6"));
       if (!library)
         return false;
 
